interfaces/datastructure/Frame.h

```diff
@@ -90,6 +90,12 @@ class SOLARFRAMEWORK_API Frame {
 	/// @brief set keypoints
 	/// @param[in] kpts: keypoints
     void setKeypoints(const std::vector<Keypoint>& kpts);
+	
+    /// @brief update keypoint class id
+    /// @param[in] i: index of keypoint
```

Comment:
In Keypoint.h you didn't use colon ':' in your parameter documentation. I'd suggest you remove them too here. Doxygen generates the same output with or without colon, but if a space is added by mistake between the parameter name and the colon, then the output is wrong (the colon appears in the generated HTML doc).
So it's safer to simply omit them, IMHO.

Reply:
have just removed ':'